src/main/scala/com/stratio/hermes/actors/HermesClientActor.scala

```diff
@@ -72,7 +72,10 @@ class HermesClientActor(implicit config: Config) extends Actor with ActorLogging
         HermesSupervisorActor.Start(nodeIds, HermesConfig(hermesConfig, kafkaConfig, template, avroConfigOption)))
     }).getOrElse({
       //scalastyle:off
-      println("Error: To start nodes is necessary to set hermes, kafka and a template configuration.")
+      println(s"Error: To start nodes is necessary to set " +
+        s"${if(hermesConfigOption.isEmpty)"hermes"}, " +
```

Comment:
Mmm...not sure whether this would work as expected. I think that even the config is empty for one of these values the literal hermes, kafka & template will always show up. I suggest to extract this logic to a method and add a unit test to make sure that this would work fine.